html/pfappserver/lib/pfappserver/Form/Config/Provisioning/ipsk.pm

```diff
@@ -0,0 +1,59 @@
+package pfappserver::Form::Config::Provisioning::ipsk;
+
+=head1 NAME
+
+pfappserver::Form::Config::Provisioning::ipsk - Web form for IPKS provisioner
+
+=head1 DESCRIPTION
+
+=cut
+
+use HTML::FormHandler::Moose;
+extends 'pfappserver::Form::Config::Provisioning';
+
+has_field 'ssid' =>
+  (
+   type => 'Text',
+   label => 'SSID',
+  );
+
+has_field 'psk_size' =>
+  (
+   type => 'PosInteger',
+   default => 8,
+   label => 'PSK length',
+   tags => { after_element => \&help,
+             help => 'This is the length of the PSK key you want to generate. The minimum length is eight characters.' },
```

Comment:
Isn't there a way to specify a minimum attribute in PosInteger, that way we prevent human errors

Reply:
I did it in the provisioner , so if yoy set 7 it will be 8.